code/formatters/cyclonedx.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020-2023 VMware, Inc.
+
+import json
+import uuid
+from typing import List
+from packageurl import PackageURL
+from cyclonedx.model.bom import Bom
+from cyclonedx.output import get_instance
+from cyclonedx.output.xml import XmlV1Dot4
+from cyclonedx.output.json import JsonV1Dot4
+from models import DocumentReport, ExtractedDependency
+from cyclonedx.model.component import Component, ComponentType
+from formatters.base import BaseFormatter, AVAILABLE_FORMATTERS
+
+
+class CycloneDXBaseFormatter(BaseFormatter):
```

Comment:
The formatter is missing handling of findings. These should be included in the `vulnerabilities` section of the document